src/pages/iou/request/step/IOURequestStepMerchant.js

```diff
@@ -26,29 +30,42 @@ const propTypes = {
     /** Onyx Props */
     /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
     transaction: transactionPropTypes,
+
+    /** The draft transaction that holds data to be persisted on the current transaction */
+    splitDraftTransaction: transactionPropTypes,
 };
 
 const defaultProps = {
     transaction: {},
+    splitDraftTransaction: {},
 };
 
 function IOURequestStepMerchant({
     route: {
-        params: {transactionID, backTo},
+        params: {transactionID, reportID, backTo, action, iouType},
     },
-    transaction: {merchant, participants},
+    transaction,
+    splitDraftTransaction,
 }) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     const {inputCallbackRef} = useAutoFocusInput();
+
+    // In the split flow, when editing we use SPLIT_TRANSACTION_DRAFT to save draft value
+    const isEditingSplitBill = iouType === CONST.IOU.TYPE.SPLIT && action === CONST.IOU.ACTION.EDIT;
+
```

Comment:
Per our style guide, I think you should remove all the blank lines that aren't preceded by comments